apps/roam/src/utils/initializeObserversAndListeners.ts

```diff
@@ -45,6 +45,16 @@ import {
 } from "~/utils/renderTextSelectionPopup";
 import { renderNodeTagPopupButton } from "./renderNodeTagPopup";
 
+let discourseNodes: DiscourseNode[] = [];
```

Comment:
Could we reuse `configTreeRef.nodes` from `discourseConfigRef.ts` and `refreshConfigTree` from `refreshConfigTree.ts` for this?

Reply:
Ok this was me being a noob all the node data is in children

Reply:
Ok this was me being a noob all the node data is in children